manifests/deis-workflow-rc.yml

```diff
@@ -53,3 +59,9 @@ spec:
         - name: minio-user
           secret:
             secretName: minio-user
+        - name: django-secret-key
+          secret:
+            secretName: django-secret-key
+        - name: builder-key-auth
+          secret:
+            secretName: builder-key-auth
```

Comment:
Nitpick, but it seems we usually include a blank line at the end of each file to avoid the ugly red mark.